src/php/Interop/PhelCaller.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Interop;
+
+use Phel\Runtime\RuntimeFactory;
+
+final class PhelCaller implements PhelCallableInterface
+{
+    /**
+     * @param mixed[] $arguments
+     *
+     * @return mixed
+     */
+    public function callPhel(string $namespace, string $definitionName, ...$arguments)
+    {
+        $fn = $this->getPhelDefinition($namespace, $definitionName);
+
+        return $fn(...$arguments);
+    }
+
+    /**
+     * @return mixed
+     */
+    private function getPhelDefinition(string $namespace, string $definitionName)
+    {
+        $rt = RuntimeFactory::getInstance();
+        $rt->loadNs($namespace);
+
+        return $GLOBALS['__phel'][$namespace][$definitionName];
```

Comment:
Don't we have a constant for `'__phel'`?🤔

Reply:
No. But we should create one.